js/samples/advanced-csharp/Pages/Options.cshtml

```diff
@@ -0,0 +1,141 @@
+        <p><code>parent</code> Immersive reader is launched in the specified element, otherwise will be launched in the main document body.</p>
+        <h3>Examples:</h3><br />
+        <section class='section' id='SampledisableLanguageDetection'>
+            <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('disableLanguageDetection')"></div>
+            Specify the content language: <input id='langdisableLanguageDetection' value='en' /> (i.e 'fr', 'es', 'de', 'zh')
+            <h3 id='titledisableLanguageDetection'>Geography (Speech, Syllables, Parts of Speech and Picture Dictionary are all affected by the language.)</h3>
+            <p id='textdisableLanguageDetection'>The study of Earth's landforms is called physical geography. Landforms can be mountains and valleys. They can also be glaciers, lakes or rivers. Landforms are sometimes called physical features. It is important for students to know about the physical geography of Earth. The seasons, the atmosphere and all the natural processes of Earth affect where people are able to live. Geography is one of a combination of factors that people use to decide where they want to live.</p>
+        </section>
+        <div id='Cols'>
+            <section class='section' id='SampledisableGrammar'>
+                <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('disableGrammar')"></div>
+                <h3 id='titledisableGrammar'>Water (Parts of Speech, Syllables and Picture Dictionary are disabled - <span id='langdisableGrammar'>en</span>)</h3>
+                <p id='textdisableGrammar'>Fresh water sources also influence where people settle. People need water to drink. They also need it for washing. Throughout history, people have settled near fresh water. Living near a water source helps ensure that people have the water they need.</p>
+            </section>
+            <section class='section' id='SampledisableTranslation'>
+                <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('disableTranslation')"></div>
+                <h3 id='titledisableTranslation'>Mountain Ranges (Translation disabled - <span id='langdisableTranslation'>en</span>)</h3>
+                <p id='textdisableTranslation'>The physical features of a region are often rich in resources. Within a nation, mountain ranges become natural borders for settlement areas. In the U.S., major mountain ranges are the Sierra Nevada, the Rocky Mountains, and the Appalachians.</p>
+            </section>
+        </div>
+        <section class='section' id='Sampleparent'>
+            <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('parent')"></div>
+            <input type="checkbox" id="checkboxparent" name="Parent" value="true">
+            <label for="vehicleparent"> Launch in Parent </label><br>
```

Comment:
vehicleparent? you mean checkboxparent?

Also, use camel case

Reply:
correct, done